packages/internal/bridge/sdk/src/lib/utils.ts

```diff
@@ -72,6 +72,52 @@ export function isWrappedIMX(token: FungibleToken, source: string) {
   return token.toUpperCase() === getWrappedIMX(source).toUpperCase();
 }
 
+/**
+ * Assumes the destination chain is correct (i.e. it doesn't check the destination chain)
+ */
+export function isDeposit(sourceChainId: string, bridgeInstance: BridgeInstance): boolean {
+  return sourceChainId === bridgeInstance.rootChainID;
+}
+
+/**
+ * Assumes the destination chain is correct (i.e. it doesn't check the destination chain)
+ */
+export function isWithdraw(sourceChainId: string, bridgeInstance: BridgeInstance): boolean {
+  return sourceChainId === bridgeInstance.childChainID;
+}
+
+export function isWithdrawNotWrappedIMX(
+  token: FungibleToken,
+  sourceChainId: string,
+  bridgeInstance: BridgeInstance,
+): boolean {
+  return isWithdraw(sourceChainId, bridgeInstance) && !isRootIMX(token, sourceChainId);
+}
+
+export function isWithdrawWrappedIMX(
+  token: FungibleToken,
+  sourceChainId: string,
+  bridgeInstance: BridgeInstance,
+): boolean {
+  return isWithdraw(sourceChainId, bridgeInstance) && isRootIMX(token, sourceChainId);
```

Comment:
can i confirm that checking `isRootIMX()` against `token` here correct? I would have assumed `token` here would be the address on L2, while the `isRootIMX()` fn compares against address on L1?